docs/csharp/language-reference/compiler-options/deterministic-compiler-option.md

```diff
@@ -8,7 +8,7 @@ helpviewer_keywords:
   - "deterministic compiler option [C#]"
   - "/deterministic compiler option [C#]"
 ---
-# -deterministic
+# -deterministic compiler option (C#)
```

Comment:
Do you want me to add this suffix to all compiler options for consistency?

Reply:
yes